### ISSUE TITLE
Error in backward

### ISSUE THREAD

**Issue by xgniu** [automan000/Convolution_LSTM_PyTorch#2]
Thanks for your implementation of conv-lstm. However, there may exist some bugs in the code. I use the code as a part of my project. The convolutional features are extracted from images and are passed to the conv-lstm. Following the conv-lstm are a fully-connected layer and the loss. But the loss.backward() will report an error, which tells that the 'retain_graph' parameter should be true. However, setting retain_graph=True will consume more and more memory and slow down the program.

**Comment by automan000** [comment]
Hello, I cannot reproduce this bug. Could you provide the certain part of your code?
